python/ray/projects/scripts.py

```diff
@@ -178,99 +286,38 @@ def stop():
         "If set, run the command as a raw shell command instead of looking up "
         "the command in the project config"),
     is_flag=True)
-def start(command, args, shell):
-    project_definition = load_project_or_throw()
-
-    if shell:
-        command_to_run = command
+def session_start(command, args, shell):
+    runner = SessionRunner(command, args, shell)
+    if shell or command:
+        logger.info("[1/4] Creating cluster")
```

Comment:
To reduce duplication here, maybe just set the total number of steps in the if statement?

Reply:
done